src/fuzz_search.c

```diff
@@ -0,0 +1,39 @@
+#include "vim.h"
+
+#ifndef PROTO
+
+    int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    static int initialized = 0;
+    if (!initialized) {
+	initialized = fuzzer_init();
+        if (!initialized)
+            abort();
+    }
+
+    // Input format:
+    // The first line is regex pattern
+    // The rest is the text to search.
+
+    void *p = memchr(data, '\n', size);
+    // At least two lines.
+    if (!p || p == data)
```

Comment:
Since `data` content will be mutated by libfuzzer, I suppose that '\n' may not always be present in `data`.
In such case, we could perhaps use the full string for the regex, and the text may have a default constant string?

Otherwise the fuzzer waste many runs on mutated input that end up doing nothing.

Reply:
how about searching empty text if only one line?

pseudo code:
if (only one line) {
   `:call setline(2, "")`
}

Reply:
done